### ISSUE TITLE
Buckets for the infra pipeline create in step 4 should used the default region provided by the user

### ISSUE THREAD

**Issue by daniel-cit** [terraform-google-modules/terraform-example-foundation#666]
### TL;DR

The Buckets created for for the infra pipeline create in step 4 :

- Terraform state bucket
- Cloud build artifacts bucket
- Cloud build default bucket

should used the default region provided by the user

### Expected behavior

The three buckets
- Terraform state bucket
- Cloud build artifacts bucket
- Cloud build default bucket

Should be create in the `default_region` provided by the user

### Observed behavior

Buckets
- Terraform state bucket
- Cloud build artifacts bucket

are always created in the region  `us-central1`

And the Cloud build default bucket (source staging bucket) is create in the multi-region `us`

### Terraform Configuration

```hcl
repo was deployed using the defaults values but changing the regions so that they are in Europe.
```

### Terraform Version

```sh
Terraform v1.0.6
on linux_amd64
```

### Additional information

The  Cloud build default bucket is created automatically in the `us` location when the cloud build API is used by the first time.
A bucket with the name `<PROJECT_ID>_cloudbuild` can be created in the selected region before the creation of the first build.
This bucket will then be used by cloud build.
